bitnami/zookeeper/values-production.yaml

```
@@ -258,6 +257,18 @@ securityContext:
   fsGroup: 1001
   runAsUser: 1001
 
+## Add initContainers to the web pods.
+## Example:
+## initContainers:
+##   - name: your-image-name
+##     image: your-image
+##     imagePullPolicy: Always
+##     ports:
+##       - name: portname
+##         containerPort: 1234
+##
+initContainers:
```

Using the previous suggested approach
```suggestion
initContainers: []
```